modules/virt-change-vm-instance-type-cli.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/creating_vm/virt-creating-vms-from-instance-types.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-change-vm-instance-type-cli_{context}"]
+= Changing the VM instance type by using the CLI
+
+[role="_abstract"]
+As a VM owner, if you want to change the instance type of a VM, you can do this by changing the `name` field in the VM spec. This triggers the update logic, ensuring a new, immutable controller revision snapshot is taken of the new resource configuration.
```

Comment:
If reasonably possible, I'd try to weave in some example usecases/reasons for changing an instance type. Alternatively, it could also be added in the higher-level assembly abstract ("Changing the instance type for a VM"). As a result, this would much better conform to JTBD requirements.

Reply:
Addressing at the assembly level